docs/reporting.md

```diff
@@ -57,42 +57,49 @@ The general information can be useful for MFTF contributors, but can be ignored
-Powered by PHPUnit 6.5.13 by Sebastian Bergmann and contributors.
+Generate Tests Command Run
+Codeception PHP Testing Framework v4.1.4
+Powered by PHPUnit 9.1.3 by Sebastian Bergmann and contributors.
+Running with seed: 
 
 Magento\FunctionalTestingFramework.functional Tests (2) ------------------------
-Modules: \Magento\FunctionalTestingFramework\Module\MagentoWebDriver, \Magento\FunctionalTestingFramework\Helper\Acceptance, \Magento\FunctionalTestingFramework\Helper\MagentoFakerData, \Magento\FunctionalTestingFramework\Module\MagentoRestDriver, PhpBrowser, \Magento\FunctionalTestingFramework\Module\MagentoSequence, \Magento\FunctionalTes
+Modules: \Magento\FunctionalTestingFramework\Module\MagentoWebDriver, \Magento\FunctionalTestingFramework\Module\MagentoSequence, \Magento\FunctionalTestingFramework\Module\MagentoAssert, \Magento\FunctionalTestingFramework\Module\MagentoActionProxies, Asserts, \Magento\FunctionalTestingFramework\Helper\HelperContainer
 ```
 
-After the test generation command (mentioned in the previous section), MFTF delegates control to the `vendor/codeception` tool, which is the `Codeception PHP Testing Framework` of version `2.3.9` that uses `PHPUnit` of version `6.5.13`.
+After the test generation command (mentioned in the previous section), MFTF delegates control to the `vendor/codeception` tool, which is the `Codeception PHP Testing Framework` of version `4.1.4` that uses `PHPUnit` of version `9.1.3`.
 
 The tool runs `2 Tests` using the configuration defined in the `functional` suite under the `Magento\FunctionalTestingFramework` namespace.
 The corresponding configuration file is `acceptance/tests/functional.suite.yml`.
-It enables `Modules: \Magento\FunctionalTestingFramework\Module\MagentoWebDriver, \Magento\FunctionalTestingFramework\Helper\Acceptance, \Magento\FunctionalTestingFramework\Helper\MagentoFakerData, \Magento\FunctionalTestingFramework\Module\MagentoRestDriver, PhpBrowser, \Magento\FunctionalTestingFramework\Module\MagentoSequence, ...`
+It enables `Modules: \Magento\FunctionalTestingFramework\Module\MagentoWebDriver, \Magento\FunctionalTestingFramework\Module\MagentoSequence, \Magento\FunctionalTestingFramework\Module\MagentoAssert, \Magento\FunctionalTestingFramework\Module\MagentoActionProxies, Asserts, \Magento\FunctionalTestingFramework\Helper\HelperContainer,..`
 
 #### Passed tests
 
 The next chunk of the log reports about test execution of the first test:
 
 ```terminal
-AdminLoginTestCest: Admin login test
-Signature: Magento\AcceptanceTest\_default\Backend\AdminLoginTestCest:AdminLoginTest
-Test: tests/functional/Magento/FunctionalTest/_generated/default/AdminLoginTestCest.php:AdminLoginTest
+AdminLoginSuccessfulTestCest: Admin login successful test
+Signature: Magento\AcceptanceTest\_default\Backend\AdminLoginSuccessfulTestCest:AdminLoginSuccessfulTest
+Test: tests/functional/tests/MFTF/_generated/default/AdminLoginSuccessfulTestCest.php:AdminLoginSuccessfulTest
```

Comment:
Is this path based on standalone setup? Maybe we should keep using the embedded path.

Reply:
fixed